setup.py

```diff
@@ -43,7 +43,7 @@
     zip_safe=False,
     tests_require=TEST_REQUIREMENTS,
     dependency_links=[
-        "http://github.com/divio/django-cms/tarball/release/4.0.x#egg=django-cms-4.0.0",
+        "https://github.com/jonathan-s/django-cms/tarball/django-2.2#egg=django-cms-4.0.0",
```

Comment:
Needs to be replaced with the following once again when https://github.com/divio/djangocms-versioning/pull/199 is merged. 

```suggestion
        "http://github.com/divio/django-cms/tarball/release/4.0.x#egg=django-cms-4.0.0",
```